pdsfile/pds4file/rules/cassini_iss.py

```
@@ -286,6 +286,160 @@
+
+# Map a logical path of an archive file name to a list of logical paths of the included
+# directories
+archive_dirs = translator.TranslatorByRegex([
+    ### cassini_iss_cruise ###
+    # bundle xml, non browse_raw or data_row collections
+    (r'.*archives-(.*/cassini_iss)/(cassini_iss_cruise)/bundle_xml_non_data_browse_collections.tar.gz', 0,
+        [r'\1/\2/bundle.xml',
+         r'\1/\2/document',
+         r'\1/\2/xml_schema',
+         r'\1/\2/context']
+    ),
```

Flake8 (7.2.0)

[error] closing bracket does not match visual indentation

(E124)